internal/config/default.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"github.com/snapp-incubator/nats-blackbox-exporter/internal/client"
+	"github.com/snapp-incubator/nats-blackbox-exporter/internal/logger"
+)
+
+// Default return default configuration.
+// nolint: mnd
+func Default() Config {
```

Comment:
you can use this package too
https://github.com/mcuadros/go-defaults
it uses default values in tag of struct and its very neat and easy to use

Reply:
Thanks for your suggestion. I wasn't aware of this useful package. However, I'm a bit unsure about using it in this struct because the definitions of different parts of this struct are in different packages. Using this package might prevent us from having a unified default configuration in one file. What are your thoughts on this?

Reply:
I think this package works in nested structs as well but i haven't tested it. but in general current approach is good too.